src/features/user/UserProfile/components/SettingsModal.tsx

```diff
@@ -30,7 +30,9 @@ export default function SettingsModal({
 
   const logoutUser = () => {
     localStorage.removeItem('userInfo');
-    logout();
+    if (typeof window !== 'undefined') {
+      logout({returnTo:window.location.href});
+    }
```

Comment:
There a more place in our code doing this kind of logout() - @harshvitra do we need to adapt them all like this?

Reply:
Only if it works, right now it says that we need to add this Log out URL inside Auth0 dashboard